wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Clicks card

Fixes #10964 

To test:

---
- On a site with Clicks, go to Stats > Period > Clicks.
  - Tip: en.blog should do ya.
- Verify the initial view is:

<img width="350" alt="clicks_init" src="https://user-images.githubusercontent.com/1816888/52450689-61f23f80-2af9-11e9-91de-54869cd64b23.png">

---
- Select a row that does _not_ expand.
- Verify a web view is displayed.

---
- Select an expandable row.
- Verify:
  - Up to 10 child rows are displayed.
  - The label text color is the same as the data label color.
    - NOTE: this applies to child rows for _all_ stats.
  - Selecting a row displays a web view.

<img width="350" alt="clicks_expanded" src="https://user-images.githubusercontent.com/1816888/52450737-9a921900-2af9-11e9-9f72-2a9dce0184a5.png">
